src/components/Slider/Slider.js

```diff
@@ -417,13 +407,16 @@ export default class Slider extends PureComponent {
               onChange={this.handleChange}
             />
           </div>
+          <span className={`${prefix}--slider__range-label`}>
+            {formatLabel(min, minLabel)}
+          </span>
           <span className={`${prefix}--slider__range-label`}>
             {formatLabel(max, maxLabel)}
           </span>
           {!hideTextInput && (
             <TextInput
               type={inputType}
-              id="input-for-slider"
+              id={`input-for-slider${id}`}
```

Comment:
We tend to use `${id}` as a prefix instead of a suffix (e.g. https://github.com/IBM/carbon-components-react/blob/v6.39.0/src/components/Select/Select.js#L31) - Wondering if it'd make sense to align the convention.